.pulumi-java-gen.version

```diff
@@ -1 +1 @@
-1.1.0
\ No newline at end of file
+1.3.0
```

Comment:
A little strange that this is bundled with the bridge upgrade, but I guess other language SDKs are also upgraded here by-design so that makes sense possibly.